Tools/WebKitTestRunner/TestController.cpp

```diff
@@ -3489,6 +3497,11 @@ void TestController::clearAppPrivacyReportTestingData()
 
 #endif // !PLATFORM(COCOA)
 
+void TestController::setPageScaleFactor(float scaleFactor, int x, int y, CompletionHandler<void(WKTypeRef)>&& completionHandler)
+{
+    WKPageSetPageScaleFactorForTesting(mainWebView()->page(), scaleFactor, WKPoint { static_cast<double>(x), static_cast<double>(y) }, completionHandler.leak(), adoptAndCallCompletionHandler);
```

Comment:
WKPoint constructor takes a float, not a double. Just remove the `static_cast`.
```suggestion
    WKPageSetPageScaleFactorForTesting(mainWebView()->page(), scaleFactor, WKPoint { x, y }, completionHandler.leak(), adoptAndCallCompletionHandler);
```

Reply:
that is not true. see https://github.com/WebKit/WebKit/blob/8cf3d3cdb4d73ecfd90b0e44d60fec1053c2df48/Source/WebKit/Shared/API/c/WKGeometry.h#L35
It takes a double.

Reply:
int can be implicitly converted to a double in this case.